### DISCUSSION TITLE
Help needed for creation or correction of README in other languages

### DISCUSSION THREAD

**Discussion by DrewThomasson** [DrewThomasson/ebook2audiobook#14] [Q&A]
If anyone is willing to help it would be greatly appreciated. 🥺

**Comment by DrewThomasson** [comment]
Asking as I've realized it's weird to have a multilingual app that only has a readme in English

**Comment by DrewThomasson** [comment]
Chinese readme... so I don't forget lol

https://github.com/DrewThomasson/ebook2audiobookXTTS/issues/18#issuecomment-2404330803

**Comment by SuperBoss9** [comment]
May do translation of the readme.md to Russian.

**Reply by DrewThomasson** [reply]
Sounds Awesome! 👍✨

**Reply by SuperBoss9** [reply]
[README_RU.md](https://github.com/user-attachments/files/17757915/README_RU.md)
Here it is

**Reply by DrewThomasson** [reply]
Added and credited you in the commit! 

Thank you so much for helping out! 👍 ✨ 

https://github.com/DrewThomasson/ebook2audiobookXTTS/commit/56f4f65418adc8fb745bd9c742ad153dca73ac2b

**Comment by ROBERT-MCDOWELL** [comment]
@DrewThomasson Should the README be updated on some parts?

**Reply by DrewThomasson** [reply]
Correct, because of the big changes we are making with the v2.0, 

That will be necessary lol 😅
